com.unity.netcode.gameobjects/Tests/Runtime/Metrics/NetworkObjectMetricsTests.cs

```diff
@@ -189,6 +189,72 @@ public IEnumerator TrackMultipleNetworkObjectDestroySentMetric()
             Assert.AreEqual(1, objectDestroyedSentMetricValues.Select(x => x.BytesCount).Distinct().Count());
             Assert.That(objectDestroyedSentMetricValues.Select(x => x.BytesCount), Has.All.Not.EqualTo(0));
         }
+
+#if MULTIPLAYER_TOOLS_1_0_0_PRE_7
+        [UnityTest]
+        public IEnumerator TrackNetworkObjectCountAfterSpawnOnServer()
+        {
+            SpawnNetworkObject();
+
+            var waitForGaugeValues = new WaitForGaugeMetricValues(ServerMetrics.Dispatcher, NetworkMetricTypes.NetworkObjects);
+
+            yield return s_DefaultWaitForTick;
+            yield return waitForGaugeValues.WaitForMetricsReceived();
+
+            var value = waitForGaugeValues.AssertMetricValueHaveBeenFound();
+            Assert.AreEqual(3, value);
+        }
+
+        [UnityTest]
+        public IEnumerator TrackNetworkObjectCountAfterSpawnOnClient()
+        {
+            SpawnNetworkObject();
+
+            //Required a condition on client due to small delay between spawning on server and client
+            var waitForGaugeValues = new WaitForGaugeMetricValues(ClientMetrics.Dispatcher, NetworkMetricTypes.NetworkObjects, metric => (int)metric != 2);
```

Comment:
Can you help me understand the `(int)metric != 2` part?

Reply:
The `SingleClientMetricTestBase` setup spawn 2 network objects by default. I'll add that comment to make sure it's clear. Since we're expecting a change to the count, we want when the count is different than the default two.